integration/example.go

```diff
@@ -0,0 +1,36 @@
+package integration
+
+import (
+	"crypto/tls"
+	"net/http"
+
+	"github.com/diskordanz/darksky/request"
+	"github.com/shawntoffel/darksky"
+	"github.com/sirupsen/logrus"
+)
+
+type Example struct {
```

Comment:
same here: not `Example` but `Darksky` and it belongs to both struct & file names